src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -246,7 +246,13 @@ impl CopyInterpreter {
         }
 
         let mut stage_table_info = stage_table_info.clone();
-        let mut all_source_file_infos = StageTable::list_files(&stage_table_info).await?;
+        let max_files = stage_table_info.stage_info.copy_options.max_files;
+
+        let mut all_source_file_infos = if force {
```

Comment:
Why to check `force` here?

Reply:
if force = false, will into color_copied_files, limit file in color_copied_files.

if force = true, will not into color_copied_files, limit file in list_files.

Reply:
Seems it's no relation with force